content/docs/resiliency/deployment.md

```diff
@@ -65,6 +67,7 @@ To install CSM for Resiliency with the driver, the following changes are require
 | arrayConnectivityPollRate | Optional | The minimum polling rate in seconds to determine if the array has connectivity to a node. Should not be set to less than 5 seconds. See the specific section for each array type for additional guidance. | controller & node |
 | arrayConnectivityConnectionLossThreshold | Optional | Gives the number of failed connection polls that will be deemed to indicate array connectivity loss. Should not be set to less than 3. See the specific section for each array type for additional guidance. | controller |
 | driver-config-params | Required | String that set the path to a file containing configuration parameter(for instance, Log levels) for a driver.  | controller & node |
+| ignoreVolumelessPods | Optional | Boolean value that if set to true will enable CSM for Resiliency to ignore volumeless pods even if those are labeled. | controller & node |
```

Comment:
Maybe 'volumeless pods' should be replaced with  'pods without PVs attached'? "even if those are labeled" does this mean the podmon taint?